Formula/monit.rb

```diff
@@ -20,10 +20,36 @@ def install
                           "--localstatedir=#{var}/monit",
                           "--sysconfdir=#{etc}/monit",
                           "--with-ssl-dir=#{Formula["openssl@1.1"].opt_prefix}"
+    system "make"
     system "make", "install"
     etc.install "monitrc"
   end
 
+  plist_options :manual => "monit -c #{HOMEBREW_PREFIX}/etc/monitrc"
+
+  def plist
+    <<~EOS
+      <?xml version="1.0" encoding="UTF-8"?>
+      <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+      <plist version="1.0">
+        <dict>
+          <key>Label</key>              <string>#{plist_name}</string>
+          <key>ProcessType</key>        <string>Adaptive</string>
+          <key>Disabled</key>           <false/>
+          <key>RunAtLoad</key>          <true/>
+          <key>LaunchOnlyOnce</key>     <false/>
+          <key>ProgramArguments</key>
+          <array>
+            <string>#{opt_bin}/monit</string>
+            <string>-I</string>
```

Comment:
You're missing this in the `:manual` command

Reply:
Done.